src/main/java/com/erigitic/commands/AdminPayCommand.java

```diff
@@ -81,18 +85,14 @@ public CommandResult execute(CommandSource src, CommandContext args) throws Comm
                 messageValues.put("recipient", recipient.getName());
                 messageValues.put("amount", amountText.toPlain());
 
-                if (!amountStr.contains("-")) {
-                    src.sendMessage(messageManager.getMessage("command.adminpay.send.sender", messageValues));
-
-                    if (recipient.isOnline()) {
-                        recipient.getPlayer().get().sendMessage(messageManager.getMessage("command.adminpay.send.recipient", messageValues));
-                    }
-                } else {
-                    src.sendMessage(messageManager.getMessage("command.adminpay.remove.sender", messageValues));
+                String invokerMessageId = amountStr.contains("-") ? "command.adminpay.remove.sender" : "command.adminpay.send.sender";
+                String recipientMessageId = amountStr.contains("-") ? "command.adminpay.remove.recipient" : "command.adminpay.send.recipient";
 
-                    if (recipient.isOnline()) {
-                        recipient.getPlayer().get().sendMessage(messageManager.getMessage("command.adminpay.remove.recipient", messageValues));
-                    }
+                src.sendMessage(messageManager.getMessage(invokerMessageId, messageValues));
+                if (recipient.isOnline()) {
+                    recipient.getPlayer()
+                            .orElseThrow(() -> new TERuntimeException("Recipient is only but is not a player!"))
```

Comment:
Error message is worded a bit weird here

```suggestion
                            .orElseThrow(() -> new TERuntimeException("Recipient is not a player!"))
```